wp-content/themes/core/blocks/tribe/post-type-name/edit.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * React hook that is used to mark the block wrapper element.
+ * It provides all the necessary props like the class name.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-block-editor/#useblockprops
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * Server-side rendering of the block in the editor view
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-server-side-render/
+ */
+import ServerSideRender from '@wordpress/server-side-render';
+
+/**
+ * The edit function describes the structure of your block in the context of the
+ * editor. This represents what the editor will render when the block is used.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/block-api/block-edit-save/#edit
+ *
+ * @return {WPElement} Element to render.
+ */
+export default function Edit() {
+	const blockProps = useBlockProps();
+
+	return (
+		<div { ...blockProps }>
+			<ServerSideRender block="tribe/post-type-name" />
```

Comment:
Nice way to minimize the code required here. Curious: is an edit.js / `editorScript` prop required for blocks? If we omitted it, would it use the render for both contexts?

Reply:
I tested out removing the `edit.js` file from `index.js` and while the block didn't break in the editor, it didn't display anything and was just a blank block. I believe the method here is doing the same thing that the `render` attribute is doing for the FE.